ftml/misc/ffi_test.c

```diff
@@ -0,0 +1,93 @@
+#include <stdio.h>
+#include <inttypes.h>
+
+#include <ftml.h>
+
+static const char *meta_type(ftml_html_meta_type type)
+{
+	switch (type) {
+	case META_NAME:
+		return "Name";
+	case META_HTTP_EQUIV:
+		return "Http-Equiv";
+	case META_PROPERTY:
+		return "Property";
+	default:
+		/* Error */
+		return NULL;
+	}
+}
+
+int main(int argc, char **argv)
+{
+	struct ftml_html_output output;
+	struct ftml_page_info page_info = {
+		"my-page",
+		NULL,
+		"www",
+		"Test page!",
+		NULL,
+		69.0,
+		NULL,
+		0,
+		"C",
```

Comment:
Can we use C99 struct initializers here? I think the field names would make it more readable.

https://stackoverflow.com/a/330867/617159

Reply:
Oh yeah that's what that syntax was. I usually use ANSI C which would mean filling out the fields individually, but I don't really care about the standard here since it's just for a simple test.